bands/site_sections/band_admin.py

```diff
@@ -24,6 +24,21 @@ def mark_as_band(self, session, group_id):
             'message': '{} has been marked as a band'.format(group.name)
         }
 
+    @ajax
+    def remove_as_band(self, session, group_id):
+        group = session.group(group_id)
+        if not group.leader:
+            return {'message': '{} does not have an assigned group leader'.format(group.name)}
```

Comment:
From a business-logic perspective, we don't care if the group has a leader when adding or removing its band status. I'm not sure why that would be different when not using the page.